sdk-core/src/main/java/care/data4life/sdk/network/util/IHCServiceFactory.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.network.util
+
+import care.data4life.sdk.network.IHCService
+import care.data4life.sdk.network.NetworkingContract
+import care.data4life.sdk.network.typeadapter.EncryptedKeyTypeAdapter
+import com.squareup.moshi.Moshi
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+object IHCServiceFactory : NetworkingContract.IHCServiceFactory {
```

Comment:
We need to change the name as I stands for interface which is not correct and not a naming pattern we use. It could use `PhdpApi` or `HealthCloudApi` for better description of it's purpose

Reply:
Indeed...I will make a follow up.